code/controllers/subsystems/bounties.dm

```diff
@@ -0,0 +1,26 @@
+SUBSYSTEM_DEF(bounty)
+	name = "Bounty"
+	flags = SS_NO_FIRE //It's just a storage / manager object
```

Comment:
so why is it not a GLOB instead of a subsystem?

Reply:
Subsystems make it look cool and impressive 😎

It didn't occur

Reply:
I don't really like the idea of everything being a subsystem. And this? It really doesn't need to be one.